package.json

```diff
@@ -31,8 +31,8 @@
 		"require-from-string": "^2.0.2"
 	},
 	"devDependencies": {
-		"acorn": "^7.0.0",
-		"eslint-config-fnd": "^1.7.0",
+		"eslint-config-fnd": "^1.8.0",
+		"live-server": "^1.2.1",
```

Comment:
No me gusta.

You'd seperately added this dependency to the example project's `package.json`, which is where it belongs (if at all) IMO.

Note that faucet-core will already notify users if the dependency is missing: https://github.com/faucet-pipeline/faucet-pipeline-core/blob/d355e340fba67c05744fd3c3a56f305df509fef6/lib/server.js#L16

Reply:
Fixed.